desktop/apps/artwork/client/test/index.coffee

```diff
@@ -1,54 +1,77 @@
-_ = require 'underscore'
+Backbone = require 'backbone'
 benv = require 'benv'
+jsdom = require 'mocha-jsdom'
+sd = require('sharify').data
 sinon = require 'sinon'
-Backbone = require 'backbone'
+_ = require 'underscore'
 { resolve } = require 'path'
 { fabricate } = require 'antigravity'
 
+
 describe 'Artwork Client', ->
-  before (done) ->
+  before ->
+    global.OpenSeadragon = {}
+
     benv.setup =>
-      benv.expose $: benv.require('jquery'), jQuery: benv.require('jquery')
+      benv.expose
+        $: benv.require('jquery'),
+        jQuery: benv.require('jquery')
```

Comment:
`benv` injects variables into the scope of the test program. Often libraries recognize `$` as a global variable, as well as `jQuery`. By setting them both you can guard against failures.